### PR TITLE
CET changes

### PR DESCRIPTION
**Pull request by yamashi**
Contains all the CET changes required to get it to run.

### CODE REVIEW

**Comment by yamashi** [comment]
Build fails because you don't have C++20 enabled :'(

File: include/RED4ext/Scripting/Stack.hpp [hunk 1]
```diff
@@ -31,20 +31,24 @@ using StackArgs_t = std::vector<CStackType>;
 struct IStack
 {
     // clang-format off
-    virtual ~IStack() = default;                            // 00
-    virtual void* GetResultAddr() { return nullptr; };      // 08
-    virtual CBaseRTTIType* GetType() { return nullptr; };   // 10
-    virtual void sub_18(int64_t a2){};                      // 18
+    virtual ~IStack() = default;                          // 00
```
Comment (WopsS) [inline include/RED4ext/Scripting/Stack.hpp:34]: Is this intended? I remember seeing a vtbl with 4 vfuncs returning default values and the being overridden with a real one, might be a `CSomeStack : CBaseStack` and ctor inlined?

Comment (yamashi) [reply include/RED4ext/Scripting/Stack.hpp:34]: It isn't no :o

**Comment by WopsS** [comment]
Thanks!

**Pull request merged**